Modelica/Media/IdealGases/Common/Functions.mo

```diff
@@ -119,7 +119,7 @@ package Functions
 
   algorithm
     h := data.R*((-data.alow[1] + T*(data.
-      blow[1] + data.alow[2]*Math.log(T) + T*(1.*data.alow[3] + T*(0.5*data.
+      blow[1] + data.alow[2]*(if T>0 then Math.log(T) else -Modelica.Constants.inf) + T*(1.*data.alow[3] + T*(0.5*data.
```

Comment:
Should be noEvent(T>0), right?

Reply:
It is a function, right? So no need for noEvent.

Reply:
It is inside function h_Tlow

Reply:
Pushing another commit, because I did not save. The term is multiplied by T, so the opposite sign is correct.